src/frontend/get-jira-configuration.ts

```diff
@@ -50,18 +51,26 @@ const formatDate = function (date) {
 	};
 };
 
-export const getFailedConnections = (installations, subscriptions) => {
-	return installations
+export const getFailedConnections = (
+	installations: FailedInstallations[] | typeof Installation[],
+	subscriptions: Subscriptions[]
+) => {
+	return (installations as FailedInstallations[])
 		.filter((response) => !!response.error)
 		.map((failedConnection) => {
 			const sub = subscriptions.find(
-				(sub) => failedConnection.id === sub.gitHubInstallationId
-			)
+				(subscription: Subscriptions) =>
+					Number(failedConnection.id) === subscription.gitHubInstallationId
```

Comment:
pretty sure this is already a number at runtime?

Reply:
Shit, you're right. Thought I saw it as a number in the response when I checked. Let me fix the type.